org.lflang/src/org/lflang/generator/c/CGenerator.xtend

```diff
@@ -483,14 +483,19 @@ class CGenerator extends GeneratorBase {
         var coreFiles = newArrayList(
             "reactor_common.c",
             "reactor.h",
-            "pqueue.c",
-            "pqueue.h",
             "tag.h",
             "tag.c",
             "trace.h",
             "trace.c",
-            "util.h",
-            "util.c",
+            "utils/pqueue.c",
```

Comment:
It does, but reactor-c is really three different libraries: unthreaded, threaded, and federated. Copying the entirety of reactor-c files will cause a lot of unnecessary files to be copied over and bloating the `src-gen` directory.

Also, I remember that I tried to make copying of folders from classpath work a while ago, but couldn't get it to work the last time I tried. The issue was that the copy mechanism had to also work for JAR files, which was a headache to get to work right for folders. If we had such a mechanism, it would be really useful at least for our `utils`, `threaded`, and `federated` folders when they are needed. What do the other target code generators do?

Reply:
See #983.